src/SvgUtils.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2023 Inclusive Design Research Centre, OCAD University
+ * All rights reserved.
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/inclusive-design/adaptive-palette/blob/main/LICENSE
+ */
+
+import { BlissSVGBuilder } from "bliss-svg-builder";
+import { BciAvId } from "./BlissSymbol";
+import { adaptivePaletteGlobals } from "./GlobalData";
+
+/**
+ * Convert the given `BciAvId` to a SVG builder code string.  If the `BciAvId`
+ * argument is an array of BCI-AV-IDs and punctuation, concatenate the array
+ * into a string of builder code strings and punctuation marks.
+ * @param {BciAvId} - The BciAvId to convert.
+ * @return {String} - The concatenation of the builder codes and punctuation,
+ *                    e.g., "B106/B12".
+ */
+export function bciAvIdToString (bciAvId: BciAvId) {
+  let finalCode = "";
+  if (typeof bciAvId === "number") {
+    const { blissSvgBuilderCode } = bciToBlissaryId(bciAvId);
```

Comment:
When a BCI-AV-ID "1" is given to `bciToBlissaryId()` function, the function doesn't find the corresponding blissary_id and returns `undefined`. It causes this line to fail with error: `TypeError: Cannot destructure property 'blissSvgBuilderCode' of 'bciToBlissaryId(...)' as it is undefined.`.

It would be helpful to add an error handling in `bciToBlissaryId()` to report when a `bci-av-id` cannot find a match. Moreover, if `bciToBlissaryId()` returns `undefined`, the caller function should handle this return without a TypeError.